instrumentation/okhttp/okhttp-3.0/library/src/test/groovy/io/opentelemetry/instrumentation/okhttp/v3_0/OkHttp3Test.groovy

```diff
@@ -5,25 +5,23 @@
 
 package io.opentelemetry.instrumentation.okhttp.v3_0
 
+import io.opentelemetry.context.Context
 import io.opentelemetry.instrumentation.test.LibraryTestTrait
+import okhttp3.Dispatcher
 import okhttp3.OkHttpClient
 
 class OkHttp3Test extends AbstractOkHttp3Test implements LibraryTestTrait {
   @Override
   OkHttpClient.Builder configureClient(OkHttpClient.Builder clientBuilder) {
-    return clientBuilder.addInterceptor(OkHttpTracing.create(getOpenTelemetry()).newInterceptor())
+    return clientBuilder
+      // The double "new Dispatcher" style is the simplest way to decorate the default executor.
+      .dispatcher(new Dispatcher(Context.taskWrapping(new Dispatcher().executorService())))
```

Comment:
Should we mention something about this in the library instrumentation's README or Javadoc?

Reply:
Doh definitely